index.html

```diff
@@ -5081,9 +5031,60 @@ <h2 id="AudioWorkletNode">
             connected with other built-in <a><code>AudioNode</code></a>s to
             form an audio graph.
           </p>
-          <dl title=
-          "[Constructor(BaseAudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode"
-          class="idl">
+          <dl title="interface AudioWorkletNode : AudioNode" class="idl">
+            <dt>
+              Constructor(BaseAudioContext context, DOMString name, optional
+              AudioWorkletNodeOptions options)
+            </dt>
+            <dd>
+              <p>
+                The construction procedure of
+                <a><code>AudioWokletNode</code></a> and the corresponding
+                <a><code>AudioWorkletProcessor</code></a> object consists of
+                several coordinated interactions.
+              </p>
+              <p>
+                The following algorithm specifies the steps when the
+                <a><code>AudioWorkletNode</code></a> constructor invoked with
```

Comment:
"constructor invoked" -> "constructor is invoked" at least.

But maybe better phrased as

> specifies the steps in constructing an AudioWorkletNode with the given parameters.